CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 ## Unreleased
 
 IMPROVEMENTS:
+* Set failurePolicy to Fail for connectInject mutating webhook so that pods fail to schedule when the webhook is offline. [[GH-1024](https://github.com/hashicorp/consul-helm/pull/1024)]
```

Comment:
Could you also mention that we have a helm vallue to override that and the `Fail` policy is the default behavior?

Reply:
Ah great idea! thanks!